public/userscripts/manga-tracker.user.js

```diff
@@ -1890,6 +1891,37 @@ let sites = {
 		}
 	}),
 
+
+  'www.readmanga.today' : extendSite({
+		setObjVars : function() {
+			this.segments      = window.location.pathname.replace(/^(.*\/)(?:[0-9]+\.html)?$/, '$1').split( '/' );
+
+			//FIXME: Is there a better way to do this? It just feels like an ugly way of setting vars.
+			this.page_count    = $('.list-switcher-2 > li > select[name=category_type').get(0).length;
+			this.title         = this.segments[1];
+			this.chapter       = this.segments[2];
+
+			this.title_url   = this.https + '://www.readmanga.today/'+this.title+'/';
+			this.chapter_url = this.title_url + this.chapter+'/';
+
+			//Might be easier to keep chapter_url different.
+			this.chapterListCurrent = this.chapter_url.slice(0,-1);
+			this.chapterList        = generateChapterList($('.jump-menu[name=chapter_list] > option:gt(0)'), 'value');
```

Comment:
`.reverseObj()` needs to be used here so chapters are ordered from first to last.